### DISCUSSION TITLE
Whether Eclipse Theia support "one user can own or customize multiple layout"

### DISCUSSION THREAD

**Discussion by xinzhang20241216** [eclipse-theia/theia#14683] [General] [answered]
We know traditional Eclipse IDE(desktop version) support one "perspective" concept, that means one user can own or customize multiple perspective(layout) based on his/her requirement, I am not sure whether Eclipse Theia also has the similar concept like perspective, user can customize or save his/her layout.

![image](https://github.com/user-attachments/assets/dc8d346f-c7f8-432a-b9ea-bc573590d146)

**Comment by msujew** [comment, marked as answer]
Hey @xinzhang20241216,

there is currently no support in Theia for this kind of feature. See https://github.com/eclipse-theia/theia/issues/14577 for more info.

**Reply by xinzhang20241216** [reply]
Hi @msujew, thanks for your response!
